server/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "dev": "ts-node-dev --respawn src/index.js",
```

Comment:
"dev": "ts-node-dev --respawn src/index.ts",

a tvuj server je ted v souboru index.js (ne ve slozce src), to musi byt TypeScript soubor a prave ve slozce src